plugin/external/pluginEntry.go

```diff
@@ -337,18 +397,37 @@ func (e *pluginEntry) SchemaGraph() (*linkedhashmap.Map, error) {
 
 const listFormat = "[{\"name\":\"entry1\",\"methods\":[\"list\"]},{\"name\":\"entry2\",\"methods\":[\"list\"]}]"
 
+type coreEntryCreator interface {
```

Comment:
Yeah, `coreEntry` seems like a good enough name for now. Some suggestions:

* `coreEntryCreator` => `coreEntry`, `create` => `createInstance`, `coreEntryTypeID` => `coreEntries`. I think all of that reads quite well when put together. It would also help if `createInstance`'s signature included the param names, because it took me a bit to see that `pluginEntry` represented the parent (so `create(parent, decodedEntry)`)

* Maybe a separate `external/coreEntry.go` file? This could be where all the core entries go.

Reply:
Done.